src/main/java/net/minecraftforge/client/event/DimensionLightMapModificationEvent.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Minecraft Forge
+ * Copyright (c) 2016-2021.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.minecraftforge.client.event;
+
+import net.minecraft.util.math.vector.Vector3f;
+import net.minecraft.world.World;
+
+/**
+ * <p>Allows modders to adjust the lightmap colors for a specific {@code Dimension}.</p>
+ *
+ * <p>While {@code DimensionRenderInfo} could look like the closest solution for this task,
+ * its entries are stored in private, immutable map that is specific to a {@code DimensionType}.
+ * There can be many {@code Dimension}(s) per {@code DimensionType}, which means
+ * {@code DimensionRenderInfo} is limited in both functionality and encourages poor practices.</p>
+ *
+ * <p>Hence, this event is the preferred solution over DimensionRenderInfo.</p>
```

Comment:
You can just say "DimensionRenderInfo is similar but uses private, singleton maps that are not Dimension specific.".
Much less wordy.